lib/src/rewarded/rewarded.dart

```diff
@@ -106,7 +106,7 @@ enum RewardedAdEvent {
 
 class RewardItem {
   /// Returns the reward amount.
-  final int? amount;
+  final double? amount;
```

Comment:
Wouldn't num be better?

Reply:
for ios amount is double value, for android is int
double is compatible with int，i think double will be better